### ISSUE TITLE
Collector example does not work with image otel/opentelemetry-collector-contrib:0.48.0

### ISSUE THREAD

**Issue by martinxluptak** [newrelic/newrelic-opentelemetry-examples#111]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT give us access or passwords to your New Relic account or API keys! )

## Steps to Reproduce

1. clone repository
2. `cd collector/nr-otlp-export/`
3. `export NEW_RELIC_API_KEY=<my_api_key>`
4. `docker-compose -f docker-compose.yaml up`

[NOTE]: # ( Please be as specific as possible. )

## Expected Behavior

[NOTE]: # ( Tell us what you expected to happen. )

The container should start without errors.

## Relevant Logs / Console output

```
➜  nr-otlp-export git:(main) ✗ docker-compose -f docker-compose.yaml up
WARNING: The OTELCOL_ARGS variable is not set. Defaulting to a blank string.
Recreating nr-otlp-export_collector_1 ... done
Attaching to nr-otlp-export_collector_1
collector_1  | /usr/local/bin/docker-entrypoint.sh: line 363: /otelcol-contrib: No such file or directory
nr-otlp-export_collector_1 exited with code 127
```

[NOTE]: # ( Please provide specifics of the local error logs, Browser Dev Tools console, etc. if appropriate and possible. )

## Your Environment

* WSL2
* docker-compose version 1.29.2, build 5becea4c

## Additional context

Works after changing image version from `0.48.0` to `0.47.0`

[TIP]:  # ( Add any other context about the problem here. )

Unrelated note: By the way, I generally feel very lost browsing through all of the New Relic's documents on opentelemetry+New Relic integration.

**Comment by TylerHelmuth** [comment]
I think I know what the issue is.  I will take a look.

**Comment by TylerHelmuth** [comment]
I was wrong, I don't know the cause of this issue yet.  I am able to clone this repo and run the docker compose with both 0.47.0 and 0.48.0.  

Couple questions:
- Are you on Linux or Windows?  
- What are the file permissions for /usr/local/bin/docker-entrypoint.sh
- What happens if you change the command in docker-compose.yaml from `/otelcol-contrib` to `/otelcontribcol`? 

>Unrelated note: By the way, I generally feel very lost browsing through all of the New Relic's documents on opentelemetry+New Relic integration.

Sorry to hear that.  Have to seen the [setup doc](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup) yet?

**Comment by martinxluptak** [comment]
I have tried pruning the system images and retrying with both 0.47.0 and 0.49.0. Weirdly enough, both 0.47.0 and 0.49.0 seem to work for me.

I can not, unfortunately, try it with native Windows. WSL2/Ubuntu is my workstation and my issue was simply fixed by bumping the number and I have no time now to setup docker-compose on another shell. I simply thought I would report on (what I thought at the time was) a reproducible issue with the example.

I have followed the setup doc for the most part - if I may ask a side question, [this page](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/) contains a link to the CNCE Slack channel which could prove helpful to my integration process, however, it seems the workspace is closed to new accounts. Does it still exist and is it possible to join it and reach out?

**Comment by TylerHelmuth** [comment]
This isn't the first issue I've seen with 0.48.0.  I'll get the example updated to 0.49.0 today. Thanks for reporting.

> Does it still exist and is it possible to join it and reach out?

The CNCF slack is still very active and one of the best ways to interact with the OTEL community and New Relic's participation in OTEL.  [You can signup for the CNCF Slack here](https://slack.cncf.io).  Once you're signed up, the link from the New Relic docs should work (it worked for me, and I can confirm it is the most up-to-date link). If not, you can search for the channel, "otel-newrelic", withing the CNCF workspace in slack.

[Also, here is a link to the community section of the opentelemetry docs, it has other ways to engage.](https://opentelemetry.io/community/)

**Comment by TylerHelmuth** [comment]
Updated the collector example to 0.49.0.  If you follow your steps to reproduce, does it work?

**Comment by reese-lee** [comment]
@martinxluptak I just wanted to chime in to thank you for your feedback about our OpenTelemetry docs, and let you know that we are actively working on improving them by making them easier to find and surfacing more resources (such as blog post tutorials) that we have published, but are somewhat hidden. Feel free to share feedback directly in our docs site as well if there's anything specific you would like to see an improvement on, via the `Create issue` button on the top right of each of our docs.

**Comment by martinxluptak** [comment]
solved by upgrading to 0.49.0
